src/doc/src/reference/unstable.md

```diff
@@ -133,51 +132,6 @@ to any Rust tools that cargo ends up calling (like `rustc` or
 `rustdoc`). Thus, if you run `cargo -Zallow-features=`, no unstable
 Cargo _or_ Rust features can be used.
 
-### extra-link-arg
```

Comment:
Would you mind rebasing on master, and updating `unstable.md` to add a section at the bottom indicating this feature has been stabilized?  This helps with informing people which version it is available, and avoids breaking external links to that section.  Maybe something like this:

```markdown
### extra-link-arg

The `extra-link-arg` feature to specify additional linker arguments in build
scripts has been stabilized in the 1.56 release. See the [build script
documentation](build-scripts.md#outputs-of-the-build-script) for more
information on specifying extra linker arguments.
```